Feature/chat comment drafts

brodeur opened the pull request:
Take a look at drafts in pots, chats, comments!

brodeur committed:
Add My Drafts view to My Home

brodeur committed:
Merge branch 'dev' into feature/chat-comment-drafts

brodeur committed:
Drafts v1

brodeur committed:
Added a necessary missing break from merge with dev

brodeur committed:
Merge branch 'dev' into feature/chat-comment-drafts

brodeur committed:
Merge branch 'dev' into feature/chat-comment-drafts

tibetsprague committed:
Merge branch 'dev' into feature/chat-comment-drafts

tibetsprague committed:
Ignore query params when deciding if its a different post draft or not

So that you can change post type and not have it remove all post content

tibetsprague committed:
Merge branch 'dev' into feature/chat-comment-drafts

tibetsprague commented:
just tested this more and right now its not saving comment drafts correctly, and its not loading drafts from their saved version correctly always. also it seems to sometimes create duplicate drafts of a new post in the same group which shouldnt be possible

tibetsprague committed:
Merge branch 'dev' into feature/chat-comment-drafts

tibetsprague committed:
Merge branch 'dev' into feature/chat-comment-drafts

tibetsprague committed:
Merge branch 'dev' into feature/chat-comment-drafts

tibetsprague committed:
Update drafts functionality to store drafts on the server instead of on device

tibetsprague commented on apps/backend/api/graphql/makeSchema.js:
```diff
@@ -243,50 +246,7 @@ export default async function makeSchema ({ req }) {
       Mutation: makeMutations({ fetchOne }),
       Subscription: makeSubscriptions(),
 
-      // Custom Type resolvers
```
this is unrelated, just moving stuff around, long story, but is slightly cleaner

tibetsprague committed:
Add post type differentiation between drafts

If switching post types and there are drafts for both it asks you whether to switch or to keep current content

Never save empty drafts

More cleanup

tibetsprague committed:
Remove really old stuff around communities, networks and polymorphic groups

Also old follows table.
Remove mogrations from 2017 and 2018

yarn dump now works as is without any edits to the schema.sql

tibetsprague committed:
Merge branch 'clean-up-old-community-and-network-stuff' into feature/chat-comment-drafts

tibetsprague committed:
Try to fix draft not deleting after post
